internal/providers/terraform/azure/testdata/automation_job_schedule_test/automation_job_schedule_test.tf

```diff
@@ -0,0 +1,43 @@
+provider "azurerm" {
+  skip_provider_registration = true
+  features {}
+}
+
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "East US"
+}
+
+resource "azurerm_automation_job_schedule" "zeroMinutes" {
+  resource_group_name     = azurerm_resource_group.example.name
+  automation_account_name = "tf-automation-account"
+  schedule_name           = "hour"
+  runbook_name            = "Get-VirtualMachine"
+
+  parameters = {
+    resourcegroup = "tf-rgr-vm"
+    vmname        = "TF-VM-01"
+  }
+}
+resource "azurerm_automation_job_schedule" "fiveMinutes" {
+  resource_group_name     = azurerm_resource_group.example.name
+  automation_account_name = "tf-automation-account"
+  schedule_name           = "hour"
+  runbook_name            = "Get-VirtualMachine"
+
+  parameters = {
```

Comment:
This is not very important, but we can avoid `optional` config parameters (like this block), on which the cost does not depend for reduce the number of lines of code